src/_posts/2025-08-06-the-ukpa-needs-you.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+author: The UK Python Association Trustees
+title: The UKPA needs you!
+---
+There is a charitable organisation, the [UK Python Association](https://uk.python.org/) (UKPA), that makes PyCon UK possible. But what exactly is the UKPA, what else does it do, and why should you consider getting involved?
+
+## What is the UKPA?
+
+The UK Python Association is a registered charity (registered in England and Wales, charity no. [1173471](http://apps.charitycommission.gov.uk/Showcharity/RegisterOfCharities/CharityWithoutPartB.aspx?RegisteredCharityNumber=1173471&SubsidiaryNumber=0)) dedicated to advancing education in the Python programming language in the UK and beyond. Established to support and grow the Python community, the UKPA serves as the driving force behind the annual PyCon UK conference and other initiatives that promote Python education and collaboration.
```

Comment:
The charity link doesn't seem to be working. The following link seems to work better (and is https): https://register-of-charities.charitycommission.gov.uk/en/charity-search/-/charity-details/5101629?_uk_gov_ccew_onereg_charitydetails_web_portlet_CharityDetailsPortlet_organisationNumber=5101629